### PR TITLE
perfect-numbers: move test cases out of test function

### PR DESCRIPTION
**Pull request by robphoenix**

### CODE REVIEW

File: exercises/perfect-numbers/perfect_numbers_test.go [hunk 1]
```diff
@@ -19,19 +32,7 @@ func TestGivesPositiveRequiredError(t *testing.T) {
...
-		{1, ClassificationDeficient},
-		{13, ClassificationDeficient},
-		{12, ClassificationAbundant},
-		{6, ClassificationPerfect},
-		{28, ClassificationPerfect},
-		{496, ClassificationPerfect},
-		{8128, ClassificationPerfect},
-	}
-	for _, c := range cases {
+	for _, c := range testCases {
```
Comment (ferhatelmas) [inline exercises/perfect-numbers/perfect_numbers_test.go]: Since testcases aren't used by multiple tests, I'm not 100% sure about this change.

Comment (robphoenix) [inline exercises/perfect-numbers/perfect_numbers_test.go]: I  changed the test cases name so it's more obvious that they belong with a specific test.

Let me know if you're happy with this before I merge.

**Comment by robphoenix** [comment]
I changed the test cases name so it's more obvious that they belong with a specific test.

@ferhatelmas Let me know if you're happy with this before I merge.

**Comment by ferhatelmas** [comment]
Better. Good to merge.

**Pull request merged**